ganga/GangaDirac/Lib/RTHandlers/DiracRTHScript.py.template

```diff
@@ -27,5 +27,7 @@ j.setParametricInputData(###PARAMETRIC_INPUTDATA###)
 
 # submit the job to dirac
 j.setPlatform( '###PLATFORM###' )
+j.setNumberOfProcessors(minNumberOfProcessors=###MIN_PROCESSORS###, maxNumberOfProcessors=###MAX_PROCESSORS###)
+
```

Comment:
Can we just wrap this in a `try-except` clause and ignore it if the client raises an `AttributeError`

Reply:
Ah yes, good idea.

Reply:
This catches the `AttributeError` for very old Dirac versions, and a `TypeError` as the keyword arguments changed.